spk/sabnzbd-beta/Makefile

```diff
@@ -0,0 +1,58 @@
+SPK_NAME = sabnzbd-beta
```

Comment:
Keeping in line with other packages, use `sabnzbd-testing`. This should be changed for all occurrences of `sabnzbd-beta`...probably the only exception is the changelog if at all.